src/kraft.erl

```diff
@@ -45,8 +74,58 @@ start(Opts, Routes) ->
     }),
     Ref.
 
-stop(Ref) -> kraft_instance:stop(Ref).
+%%------------------------------------------------------------------------------
+-doc """
+Stops the Kraft server instance.
+""".
+-spec stop(kraft_instance:ref()) -> ok.
+stop(Ref) ->
+    kraft_instance:stop(Ref).
+
+%%------------------------------------------------------------------------------
+-doc """
+Renders a template with the given context.
+
+This function allows you to render templates programmatically outside
+of the request/response cycle. Useful for generating emails, reports,
+or other content.
+
+## Arguments
+
+- `Conn` - Kraft connection object (can be minimal)
+- `Template` - Template name (e.g., "email.html")
+- `Context` - Data context for template variables
+
+## Returns
+
+- `iodata()` - Rendered HTML content
+
+## Examples
+
+```erlang
+% Render a welcome email
+EmailHtml = kraft:render(Conn, "welcome_email.html", #{
+    name => "John",
+    activation_link => "https://example.com/activate/123"
+}).
+
+% Render a report template
+ReportHtml = kraft:render(Conn, "monthly_report.html", #{
+    month => "January",
+    data => get_monthly_data()
+}).
+```
+
+## Template Location
+
+Templates should be located in:
+`priv/web/templates/{Template}.mustache`
+
+## See Also
 
+- `kraft_template:render/3` - For advanced template operations
```

Comment:
It was some time ago, but I believe my intention is that the user should always go through `kraft:...` functions, so referring to the internal function is not really helpful here.